app/src/main/java/com/example/cartoon/MainActivity.kt

```diff
@@ -77,7 +89,10 @@ class MainActivity : ComponentActivity() {
 
         // Check if no image is selected
         if (drawable == null) {
-            Toast.makeText(this, "No image selected", Toast.LENGTH_SHORT).show()
+            FancyToast.makeText(
```

Comment:
See whether if it possible to move FancyToast to a helper function and pass just the required argument.

Reply:
Would you like to do it? You can open an issue for this and we will assign it to you. Thank you for suggestion @lingarajsankaravelu.

Reply:
why not. @ashut0sh75 . I think you approved the PR in between my review.

Reply:
Yes that was the case.